kythe/go/platform/tools/kzip/createcmd/createcmd.go

```diff
@@ -145,3 +157,15 @@ func addFile(out *kzip.Writer, cu *apb.CompilationUnit, path string) error {
 	})
 	return nil
 }
+
+func unmarshalDetails(details string, msg *[]*anypb.Any) error {
```

Comment:
Idiomatically return `([]*anypb.Any, error)` instead of using an output parameter